lib/loaders/eslint.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * This file is part of the Symfony Webpack Encore package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+const loaderFeatures = require('../features');
+
+/**
+ * @param {WebpackConfig} webpackConfig
+ * @return {Object} of options to use for eslint-loader options.
+ */
+module.exports = {
+    getOptions(webpackConfig) {
+        loaderFeatures.ensurePackagesExist('eslint');
+
+        const eslintLoaderOptions = {
+            parser: 'babel-eslint',
```

Comment:
According to their docs:

> You only need to use babel-eslint if you are using types (Flow) or experimental features not supported in ESLint itself yet. Otherwise try the default parser (you don't have to use it just because you are using Babel).

I don't think this applies to Encore users with "standard" config. If they're using experimental features, then they would need to configure some extra babel presets or plugins (which is something custom). So, I'm wondering if there's a good enough reason to use *this* parser.

Reply:
No it doesnt. I should probably check here if enableBable or enableRract thingies are called and add by default.

Reply:
Hmm. Yea.... now that I process you reply, I *do* think it should be added always :p. The reason is that there is NOT an `enableBabel()` method - it's *always* enabled. And so, even though most users won't use any special syntax, we should try to make lives easier in case they do. If we *did* allow a user to disable babel in the future, *then* we could backup to the normal linter.

Reply:
Ok, so I leave it in now?